src/Employer/Employer.Web/Orchestrators/Part2/QualificationsOrchestrator.cs

```diff
@@ -58,25 +58,42 @@ public async Task<QualificationsViewModel> GetQualificationsViewModelAsync(Vacan
             return vm;
         }
 
-        public async Task<AddQualificationViewModel> GetAddQualificationViewModelAsync(VacancyRouteModel vrm)
+        public async Task<QualificationViewModel> GetQualificationViewModelAsync(VacancyRouteModel vrm, int? editIndex)
         {
-            var vacancyTask = Utility.GetAuthorisedVacancyForEditAsync(_client, _vacancyClient, vrm, RouteNames.Qualification_Add_Get);
+            var vacancyTask = Utility.GetAuthorisedVacancyForEditAsync(_client, _vacancyClient, vrm, RouteNames.Qualification_Get);
             var allQualificationsTask = _vacancyClient.GetCandidateQualificationsAsync();
 
             await Task.WhenAll(vacancyTask, allQualificationsTask);
 
-            var vm = new AddQualificationViewModel
+            var vacancy = vacancyTask.Result;
+
+            var vm = new QualificationViewModel
             {
-                Title = vacancyTask.Result.Title,
+                Title = vacancy.Title,
                 QualificationTypes = allQualificationsTask.Result,
+                CancelRoute = vacancy.Qualifications?.Any() == true ? RouteNames.Qualifications_Get : RouteNames.Vacancy_Preview_Get,
```

Comment:
I think  ` == true` is pretty implicit in `.Any()`

Reply:
`?.Any()` so could be null so needs the explicit true to make a boolean expression.

Reply:
missed the `?`